scripts/analysis.py

```diff
@@ -250,229 +116,198 @@ def get_sim_time_duration(cursor):
-    outputname: str
-        filename of the multi line plot file
+    in_list: list
+        list of data to be created into timeseries
+        list[0] = time
+        list[1] = value, quantity
+    duration: int
+        duration of the simulation
+    multiplyby: int
+        integer to multiply the value in the list by for
+        unit conversion from kilograms
+    cumulative: boolean
+        determine whether returned timeseries should be
+        cumulative sum over time
 
     Returns
     -------
-
+    timeseries list of data
     """
-
-    cur = cursor
-    agent_ids = get_agent_ids(cur, facility)
-    if influx_bool is True:
-        resources = cur.execute(exec_string(agent_ids,
-                                            'transactions.receiverId',
-                                            'sum(quantity), time, qualid')
-                                + ' GROUP BY time, qualid').fetchall()
+    value = 0
+    value_timeseries = []
+    array = np.array(in_list)
+    if cumulative:
+        for i in range(0, duration):
+            if len(array) > 0:
+                value += sum(array[array[:, 0] == i][:, 1])
+            value_timeseries.append(value * multiplyby)
     else:
-        resources = cur.execute(exec_string(agent_ids,
-                                            'transactions.senderId',
-                                            'sum(quantity), time, qualid')
-                                + ' GROUP BY time, qualid').fetchall()
-
-    compositions = cur.execute('SELECT qualid, nucid, massfrac\
-                               FROM compositions').fetchall()
+        for i in range(0, duration):
+            value = sum(array[array[:, 0] == i][:, 1])
+            value_timeseries.append(value * multiplyby)
 
-    init_year, init_month, duration, timestep = get_sim_time_duration(cur)
-    isotope, mass, time_list = isotope_mass_time_list(resources, compositions)
+    return value_timeseries
 
-    waste_dict = get_waste_dict(isotope, mass, time_list, duration)
 
-    if influx_bool is False:
-        stacked_bar_chart(waste_dict, timestep,
-                          'Years', 'Mass [kg]',
-                          title, outputname, init_year)
-    else:
-        multi_line_plot(waste_dict, timestep,
-                        'Years', 'Mass [kg]',
-                        title, outputname, init_year)
+"""GETTERS"""
 
 
-def commodity_from_facility(cursor, facility, commodity):
-    """ Returns timeseries of commodity outflux from facility
+def snf(cursor):
+    """returns a dictionary of isotopics in sink at the end of simulation
 
     Parameters
     ----------
     cursor: sqlite cursor
         sqlite cursor
-    facility: str
-        name of facility type
-    commodity: list
-        list of commodities
 
     Returns
     -------
-    dictionary of timeseries of mass outflux of commodity from facility
+    snf_dict: dictionary
+        dirctionary with key: isotope, and value: mass
```

Comment:
- spelling = dictionary. 
- the colons are meaningful to sphinx. to avoid clashes, consider " key=isotope, and value=mass"